src/applications/personalization/va-profile/containers/VAProfileApp.jsx

```diff
@@ -29,30 +33,32 @@ class VAProfileApp extends React.Component {
           user={this.props.account}
           loginUrl={this.props.loginUrl}
           verifyUrl={this.props.verifyUrl}>
-          <DowntimeNotification appTitle="user profile page" dependencies={[services.mvi, services.emis]}>
-            <ProfileView
-              user={this.props.account}
-              fetchAddressConstants={this.props.fetchAddressConstants}
-              fetchContactInformation={this.props.fetchContactInformation}
-              fetchHero={this.props.fetchHero}
-              fetchMilitaryInformation={this.props.fetchMilitaryInformation}
-              fetchPersonalInformation={this.props.fetchPersonalInformation}
-              profile={this.props.profile}
-              message={{
-                content: this.props.profile.message,
-                clear: this.props.clearMessage
-              }}
-              updateActions={this.props.updateActions}
-              updateFormFieldActions={this.props.updateFormFieldActions}
-              modal={{
-                open: this.props.openModal,
-                currentlyOpen: this.props.profile.modal,
-                formFields: this.props.profile.formFields,
-                pendingSaves: this.props.profile.pendingSaves,
-                errors: this.props.profile.errors,
-                clearErrors: this.props.clearErrors
-              }}/>
-          </DowntimeNotification>
+          <ProfileView
+            user={this.props.account}
+            fetchAddressConstants={this.props.fetchAddressConstants}
+            fetchContactInformation={this.props.fetchContactInformation}
+            fetchHero={this.props.fetchHero}
+            fetchMilitaryInformation={this.props.fetchMilitaryInformation}
+            fetchPersonalInformation={this.props.fetchPersonalInformation}
+            profile={this.props.profile}
+            message={{
+              content: this.props.profile.message,
+              clear: this.props.clearMessage
+            }}
+            updateActions={this.props.updateActions}
+            updateFormFieldActions={this.props.updateFormFieldActions}
+            downtimeData={{
```

Comment:
This might be confusing, that this is called `downtimeData`, but it also includes actions and is not the same as the `downtimeData` prop here.